### PR TITLE
Highcharts: add missing wrap method to Static

### PR DESCRIPTION
**Pull request by cebor**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://www.highcharts.com/docs/extending-highcharts/extending-highcharts
- [ ] Increase the version number in the header if appropriate.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

**Comment by hanssens** [comment]
@cebor thanks for the PR! Just a question, though: do you have an exact link to the docs on http://api.highcharts.com/highcharts/ for this fix? 

I quickly looked at the docs, but couldn't find the wrap() function on the base of Highcharts. Perhaps I'm missing something. Also, fixing the build errors as the typescript-bot notifies would be great.

**Comment by cebor** [comment]
@hanssens Yes, i posted the link above: https://www.highcharts.com/docs/extending-highcharts/extending-highcharts

File: types/highcharts/index.d.ts [hunk 1]
```diff
@@ -6694,6 +6694,8 @@ declare namespace Highcharts {
         getOptions(): Options;
 
         map(array: any[], fn: Function): any[];
+
+        wrap(prototype: any, type: string, cb: Function): void;
```
Comment (andy-ms) [inline types/highcharts/index.d.ts]: Could you use a more specific type for `cb`, such as `(proceed: ???) => void`?

**Comment by cebor** [comment]
@andy-ms done

**Comment by andy-ms** [comment]
@cebor that causes a no-implicit-any err; you can fix it by adding `: any[]` explicitly.

**Comment by andy-ms** [comment]
Thanks!

**Pull request merged**